DevelopedCalculator.js

```diff
@@ -29,7 +29,7 @@ class DevelopedCalculator extends Calculator {
     }
     onClearButtonClick() {
         super.onClearButtonClick()
-        this.history = ""
+        this.history = [""]
```

Comment:
` [""]` is this empty array?

Reply:
I have pushed the correct one before reading this comment